### PR TITLE
#4619 Rename shabondi source and sink service's classname

### PR DESCRIPTION
**Pull request by vitojeng**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4619)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/rest/rest-shabondi.rst [hunk 1]
```diff
@@ -39,8 +39,8 @@ The following are common setting both Shabondi source and sink:
      the following forms are legal as well. 1) ``{"name": "n"}`` and 2) ``"n"``. Both forms are converted to ``{"group": "default", "name": "n"}``
 #. shabondi.class (**string**) — class name of Shabondi service, only the following two class names are legal:
 
-   - Shabondi source: ``oharastream.ohara.shabondi.source.Boot``
-   - Shabondi sink: ``oharastream.ohara.shabondi.sink.Boot``
+   - Shabondi source: ``oharastream.ohara.shabondi.ShabondiSource``
+   - Shabondi sink: ``oharastream.ohara.shabondi.ShabondiSink``
```
Comment (konekoya) [inline docs/src/rest/rest-shabondi.rst:43]: 💯

File: bin/start-service.sh [hunk 2]
```diff
@@ -52,9 +52,6 @@ else
     echo "configurator                             Ohara Configurator provides the service "
     echo "                                         for user and Ohara Manager to use."
     echo ""
-    echo "shabondi                                 Ohara Shabondi provides the service "
-    echo "                                         for user sending data through http request."
-    echo ""
     echo "manager                                  Running Ohara Manager. After run this command, you can "
```
Comment (konekoya) [inline bin/start-service.sh:55]: After run -> _After running_

Comment (vitojeng) [reply bin/start-service.sh:55]: @konekoya Good catch! I'm fix it in another PR.

**Comment by vitojeng** [comment]
not related error

**Pull request merged**